src/specsynthase/specbuilder.py

```diff
@@ -50,3 +50,6 @@ def add_spec(self, file_name):
         self._merge_part(self._parameters_key, parameters)
 
         return self
+
+    def dump(self, *args, **kwargs):
+        return yaml.safe_dump(*args, **kwargs)
```

Comment:
There is no input data to safe_dump, I think that can be yaml.safe_dump(self, *args, **kwargs). As SpecBuilder has no Representer, maybe you  may want unpack dict into another dict like dict(**self).